blockchain/cost.go

```diff
@@ -40,7 +42,7 @@ const (
 // transaction's virtual is based off it's cost, creating a discount
 // for any witness data it contains, proportional to the current
 // WitnessScaleFactor value.
-func GetTxVirtualSize(tx *btcutil.Tx) int64 {
+func GetTxVirtualSize(tx *wire.MsgTx) int64 {
```

Comment:
Instead of switching to only operate on a `*wire.MsgTx`, can you make an additional version which takes a `*btcutil.Tx` and just proxies to this function? 

Throughout the `btcd` codebase, callers either have a `*wire.MsgTx` or a `*btcutil.Tx` depending on the context, so I think a streamlined path for all callers should be created. Otherwise, in many places on would have to convert to a enw type solely for a single function call.